### PR TITLE
Silverstripe security advisories

### PR DESCRIPTION
**Pull request by spekulatius**

### CODE REVIEW

File: silverstripe/framework/SS-2015-026-1.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+branches:
+    3.0.x:
+        time:     2015-11-11 14:31:00
+        versions: [>=3.0.0,<=3.0.14]
+    3.1.x:
+        time:     2015-11-11 14:31:00
+        versions: [>3.1.0,<=3.1.16]
+    3.2.x:
+        time:     2015-11-11 14:31:00
+        versions: [<3.2.1]
```
Comment (stof) [inline silverstripe/framework/SS-2015-026-1.yaml]: this missing a lower bound, which would make 3.0.15 invalid too

File: silverstripe/framework/SS-2015-027-1.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+branches:
+    3.0.x:
+        time:     2015-11-13 10:30:00
+        versions: [>=3.0.0,<=3.0.14]
+    3.1.x:
+        time:     2015-11-13 10:30:00
+        versions: [>=3.1.0,<=3.1.16]
+    3.2.x:
+        time:     2015-11-13 10:30:00
+        versions: [<3.2.1]
```
Comment (stof) [inline silverstripe/framework/SS-2015-027-1.yaml]: same issue here

**Comment by spekulatius** [comment]
Hello @stof ,

thanks for having a look at this. I've open a pull request to fix this.

Spek

**Comment by spekulatius** [comment]
https://github.com/FriendsOfPHP/security-advisories/pull/120

**Pull request merged**